.travis.yml

```diff
@@ -11,18 +23,23 @@ env:
     - secure: "OpBwPc1GNvauageYOH3RscAa7wpZxgpmqDz15aigIKLNWzAhAtVUx0MleZ8rQeoqml6nrAvlnzuVHjKL2lVcjMPpjUis7bcQ5UAGK7tZK8x+qZNQxXmpXu8+pENwQA2yFaqt/xy7K5jFOrHJHTRxcPnyVG1yKakPWz53PPYUwbc="
     # SONA_PASS
     - secure: "Xw7rI/qlML1nD2e2XwlakkhKAWNGZKqqE+Q3ntTvFpfHryl7KLCvVzJ4LIavnL6kGJaWOgy9vlSoEWn5g9nqHSfE31C/k5pY5nTMAKiwiJzfAS+r0asKXW2gmKhwtcTBkqyLVOZLCJSPVlFRQyfBJHY+Fs0L3KWcnMQgtBlyDhU="
+  matrix:
+    - SCALAJS_VERSION=
+    - SCALAJS_VERSION=0.6.23
+    - SCALAJS_VERSION=1.0.0-M3
 
-script: admin/build.sh
-
-addons:
-  apt:
-    packages:
-      - openjdk-6-jdk
+matrix:
+  exclude:
+    - scala: 2.13.0-M3
+      env: SCALAJS_VERSION=1.0.0-M3
+    - scala: 2.12.4
+      jdk: openjdk6
+    - scala: 2.13.0-M3
+      jdk: openjdk6
 
-jdk:
-  - openjdk6
-  - oraclejdk8
-  - oraclejdk9
+script:
+  - if [[ "$TRAVIS_JDK_VERSION" == "openjdk6" && "$TRAVIS_SCALA_VERSION" =~ 2\.11\..* || "$TRAVIS_JDK_VERSION" == "oraclejdk8" && "$TRAVIS_SCALA_VERSION" =~ 2\.1[23]\..* ]]; then export RELEASE_COMBO=true; fi
```

Comment:
Is there a reason this line is outside `build.sh`?

Reply:
I guess my idea was to keep things that affect the build matrix in the travis file. But looking at it, I agree it's nicer to move it over to the bash script. Will do.